search-sentiments-map/public/style.css

```diff
@@ -26,6 +26,17 @@ i {
   width: 360px;
 }
 
+#timeline-slider-div {
+  background: #fff;
+  border: 3px solid #000;
+  display: flex;
+  display: -webkit-box;
+  margin: 10px;
+  padding: 10px;
+  width: 360px;

```

Comment:
Can you make the timeline larger? From what I saw on a deployed version it seems kind of small. Also, does having it as a map control make it smaller?

Reply:
I was going to handle the timeline formatting in a new branch after your front end changes are merged in, to try and format it so  that it looks good with the new front end paneling.

Reply:
sounds good